bundeslaender.txt

```diff
@@ -0,0 +1,17 @@
+Baden-Württemberg
+Bayern
+Berlin
+Brandenburg
+Bremen
+Hamburg
+Hessen
+Mecklenburg-Vorpommern
+Niedersachsen
+Dominikanische Republik
```

Comment:
:rofl:

Reply:
I was wondering why there were 17 lines in bundeslaender...

Reply:
Haha. No idea how that ended up there. Thanks for spotting it.